### PR TITLE
Add background info on extended SC mapping

### PR DESCRIPTION
**Pull request by WilcoFiers**
We have a number of rules that test both a "minimal" variant of a requirement, but also map to the "enhanced" variant of it. For example, contrast minimal and contrast enhanced. This can be a little confusing. An explanation of this can be added to the background section.

This is the first batch. I'd like to see if this is the right way to go with this, if so I can add more of them. Here is the complete list of rules that this would need to happen with:

- [x] Text has minimum contrast (afw4f7)
- [x] video element visual content has accessible alternative (c5a4ea)
- [x] meta viewport does not prevent zoom (b4f0c3)
- [x] Scrollable element is keyboard accessible (0ssw9k)
- [x] meta element has no refresh delay (bc659a)
- [x] Link has non-empty accessible name (c487ae) _(No change needed)_
- [x] Link in context is descriptive (5effbb)
- [x] Links with identical accessible names and context serve equivalent purpose (fd3a94)

Need for Final Call: 1 week, because the mapping is changing and this has implications for quite a few rules at once.

### CODE REVIEW

File: _rules/meta-viewport-b4f0c3.md [hunk 1]
```diff
@@ -36,15 +36,20 @@ For each test target, the `content` attribute, whose value is mapped to a list o
 
 ## Assumptions
 
+If any of the following is true, this rule can file while the [1.4.4 Resize text][sc144] and [1.4.10 Reflow][sc1410] can still be satisfied:
+
 - The [page][] has [visible][] [content][].
 - There is no other [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) available to resize the text content.
+- The [content][] is not by default rendered in a way that fits in an area of 320 by 256 [CSS pixels][], and needs to reflow to do so.
```
Comment (WilcoFiers) [inline _rules/meta-viewport-b4f0c3.md:43]: Let me know if anyone thinks this needs to be in a separate PR.

Comment (WilcoFiers) [reply _rules/meta-viewport-b4f0c3.md:43]: Changed my mind. I'm taking this out. Belongs in a separate PR.

File: _rules/meta-viewport-b4f0c3.md [hunk 2]
```diff
@@ -36,15 +36,20 @@ For each test target, the `content` attribute, whose value is mapped to a list o
 
 ## Assumptions
 
+If any of the following is true, this rule can file while the [1.4.4 Resize text][sc144] and [1.4.10 Reflow][sc1410] can still be satisfied:
```
Comment (Jym77) [inline _rules/meta-viewport-b4f0c3.md]:
```suggestion
If any of the following is false, this rule can fail while Success Criteria [1.4.4 Resize text][sc144] and [1.4.10 Reflow][sc1410] can still be satisfied:
```

File: _rules/meta-viewport-b4f0c3.md [hunk 3]
```diff
@@ -36,15 +36,20 @@ For each test target, the `content` attribute, whose value is mapped to a list o
...
 - There is no other [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) available to resize the text content.
+- The [content][] is not by default rendered in a way that fits in an area of 320 by 256 [CSS pixels][], and needs to reflow to do so.
 
 ## Accessibility Support
 
 Desktop browsers ignore the viewport `meta` element, and most modern mobile browsers either ignore it by default or have an accessibility option which will allow zooming. This rule is not relevant for desktop browsers, nor for most modern mobile browsers. Only users with older mobile browsers can experience issues tested by this rule.
 
 ## Background
 
+This rule is designed specificially for [1.4.4 Resize text][sc144], which requires that text can be resized up to 200%. Because text that can not be resized up to 200% will can not fit in an area of 320 by 256 [CSS pixels][], this rule maps to [1.4.10 Reflow][sc1410] as well. All passed examples in this rule satisfy both success criteria.
```
Comment (Jym77) [inline _rules/meta-viewport-b4f0c3.md:48]:
```suggestion
This rule is designed specificially for [1.4.4 Resize text][sc144], which requires that text can be resized up to 200%. Because text that can not be resized up to 200% can not fit in an area of 320 by 256 [CSS pixels][], this rule maps to [1.4.10 Reflow][sc1410] as well. All passed examples in this rule satisfy both success criteria.
```

File: _rules/text-contrast-afw4f7.md [hunk 4]
```diff
@@ -55,6 +55,8 @@ For each test target, the [highest possible contrast][] between the [foreground
 
 Passing this rule does not mean that the text has sufficient color contrast. If all background pixels have a low contrast with all foreground pixels, the success criterion is guaranteed to not be satisfied. When some pixels have sufficient contrast, and others do not, legibility should be considered. There is no clear method for determining legibility, which is why this is out of scope for this rule.
 
+This rule is designed specificially for [1.4.3 Contrast (Minimum)][sc143], which has the expected contrast ratio of 4.5:1 (or 3:1 for large text). Because text that fails a contrast ratio of 4.5:1 also fails a contrast ratio of 7:1, this rule maps to [1.4.6 Contrast (Enhanced)][sc146] as well. In order to adiquiatly test the [expectation](#expectation), some of the passed examples do not satisfy [1.4.6 Contrast (Enhanced)][sc146].
```
Comment (Jym77) [inline _rules/text-contrast-afw4f7.md:58]:
```suggestion
This rule is designed specificially for [1.4.3 Contrast (Minimum)][sc143], which has the expected contrast ratio of 4.5:1 (or 3:1 for large text). Because text that fails a contrast ratio of 4.5:1 also fails a contrast ratio of 7:1, this rule maps to [1.4.6 Contrast (Enhanced)][sc146] as well. In order to adequately test the [expectation](#expectation), some of the passed examples do not satisfy [1.4.6 Contrast (Enhanced)][sc146].
```

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 5]
```diff
@@ -78,6 +83,8 @@ See [Video element description track: accessibility support](https://act-rules.g
 
 ## Background
 
+This rule is designed specificially for [1.2.3 Audio Description or Media Alternative (Prerecorded)][sc123], which has the expected either audio description or a media alternative. If a video has neither, by definition it also fails for both [1.2.5 Audio Description (Prerecorded)][125] and [1.2.8 Media Alternative (Prerecorded)][sc128]. In order to adiquiatly test the [expectation](#expectation) of this rule, some of the passed examples do not satisfy [1.2.5 Audio Description (Prerecorded)][125] or [1.2.8 Media Alternative (Prerecorded)][sc128].
```
Comment (Jym77) [inline _rules/video-alternative-for-visual-c5a4ea.md:86]:
```suggestion
This rule is designed specificially for [1.2.3 Audio Description or Media Alternative (Prerecorded)][sc123], which expects either audio description or a media alternative. If a video has neither, by definition it also fails for both [1.2.5 Audio Description (Prerecorded)][125] and [1.2.8 Media Alternative (Prerecorded)][sc128]. In order to adequately test the [expectation](#expectation) of this rule, some of the passed examples do not satisfy [1.2.5 Audio Description (Prerecorded)][125] or [1.2.8 Media Alternative (Prerecorded)][sc128].
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
That looks like the good direction.

**Review by Jym77 — DISMISSED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by Jym77 — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by jeeyyy — APPROVED** [review]
